app/operations/mec_check/initiate_application_mec_checks.rb

```diff
@@ -48,7 +48,7 @@ def get_applicant_checks(application_payload, job)
         person_mec_check = get_person_mec_check(person, job, application_payload["hbx_id"])
         next unless person_mec_check.success?
         evidence = person["local_mec_evidence"]
-        evidence["request_results"] = [person_mec_check.value!]
+        evidence["request_results"] = [person_mec_check.value!.merge!(action: "Bulk Call")] # action is bulk since we only do bulk calls as of now
```

Comment:
should we get a confirmation from Dom on what he wanted to see as a display string?

Reply:
This should be fine. I just named it a bulk call since we already have a hub call.